### ISSUE TITLE
Remove domains from seq properties

### ISSUE THREAD

**Issue by rjyounes** [rjyounes/vitrolib#238]
Prevent properties from showing up on all entity types.

BUT: where are these properties supposed to show up? We will need faux properties for those entity types if we remove the domains.

@sfolsom @hudajkhan Please comment

**Comment by sfolsom** [comment]
Right, the assumption I had was that it would be driven off from the SHACL.
